website/docs/r/dnat.html.markdown

```diff
@@ -15,21 +15,38 @@ and delete destination NATs to map an external IP/port to an internal IP/port.
 
 ```hcl
 resource "vcd_dnat" "web" {
+  org              = "my-org" # Optional
+  vdc              = "my-vdc" # Optional
+  
   edge_gateway = "Edge Gateway Name"
   external_ip  = "78.101.10.20"
   port         = 80
   internal_ip  = "10.10.0.5"
   translated_port = 8080
 }
+
+resource "vcd_dnat" "forIcmp" {
+  org              = "my-org" # Optional
+  vdc              = "my-vdc" # Optional
+  
+  edge_gateway = "Edge Gateway Name"
+  external_ip  = "78.101.10.20"
+  port         = -1
+  internal_ip  = "10.10.0.5"
+  protocol = "ICMP"
```

Comment:
What about adding a remark:
`protocol = "icmp" # The value of this parameter is case-insensitive`